wakuv2/waku_test.go

```diff
@@ -599,64 +597,6 @@ func waitForEnvelope(t *testing.T, contentTopic string, envCh chan common.Envelo
 	}
 }
 
-func TestConfirmMessageDelivered(t *testing.T) {
```

Comment:
Hope we have an equivalent test in waku to validate the functionality.

Reply:
Equivalent test  is here: https://github.com/waku-org/go-waku/blob/master/waku/v2/api/publish/message_check_test.go